src/pages/index.js

```diff
@@ -70,17 +78,31 @@ import { PopupWithImage } from "../components/PopupWithImage";
     errorClass: "popup__error_visible",
   };
 
+  const avatarConfiguration = {
```

Comment:
Please, delete `avatarConfiguration` because only **one** universal `configuration` should be used for any form and validation